fissionServerSwagger.yaml

```diff
@@ -0,0 +1,386 @@
+    
+  version: 3.0.0alpha
+
+tags:
+  - name: auth
+    description: Authentication
+  - name: account
+    description: Account management
+  - name: volume
+    description: Volume management
+  - name: app
+    description: App management
+  - name: domain
+    description: Domain (DNS) management
+
+paths:
+    /auth/requestToken:
+      post:
+        tags:
+          - account
+          - auth
+        summary: Request an email verification token.
+        description: |-
+          Reqeust an email verification token, to be used to [create an account](#/paths/~1account~1create/post) or [reset the DID](#/paths/~1account~1update/post) associated with an account.
+        requestBody:
+          description: Email address
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  email:
+                    $ref: "#/components/schemas/Account/properties/email"
+        responses:
+          '200':
+            description: success
+          '400':
+            description: Bad Request
+          '429':
+            description: Too Many Requests
+          '500':
+            description: Internal Server Error
+
+    /account/create:
+      post:
+        tags:
+          - account
+        summary: Creates a new account
+        description: Creates a new Fission account.
+
+        requestBody:
+          description: Account parameters
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Account'
+
+        responses:
+          '201':
+            description: created
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - $ref: '#/components/schemas/Account'
+                    - $ref: '#/components/schemas/Volume'
+          '400':
+            description: Bad Request
+          '401':
+            description: Unauthorized
+          '500':
+            description: Internal Server Error
+        security:
+          - UCANAuth: []
+          - EmailTokenAuth: []
+
+    /account/{accountName}:
+      parameters:
+        - $ref: "#/components/parameters/AccountName"
+
+      get:
+        tags:
+          - account
+        summary: Fetches account details for the specified account.
+
+        responses:
+          '200':
+            description: Account details
+            content:
+              application/json:
+                schema:
+                  allOf:
+                    - $ref: '#/components/schemas/Account'
+                    - $ref: '#/components/schemas/Volume'
+          '400':
+            description: Bad Request
+          '500': 
+            description: Internal Server Error
+        security:
+          - UCANAuth: []
+
+    /account/{accountName}/did:
+      parameters:
+        - $ref: "#/components/parameters/AccountName"
+      post:
```

Comment:
Should we do `PATCH` methods for updates?

Reply:
Maybe! I generally just use GET and POST, but I think that's just an old habit from when you couldn't use anything else from the web. I'll change it, because it's more semantically correct, and if anyone feels strongly they'll probably let us know sooner if it's PATCH vs POST. 😅

Reply:
* I've updated it to PUT because I think that's the more semantic method for those paths.